### DISCUSSION TITLE
Help with color space please

### DISCUSSION THREAD

**Discussion by KenTanker0us** [ggarra13/mrv2#50] [Q&A]
Hi, I hope I'm writing for help in the right place.

I want to use mrv2 as a replacement for PDPlayer. I've got it installed and it's working beautifully.

My one problem is I have no idea how to configure it so EXR files will look the same as they do in PDPlayer.

Please help this lowly noob. Thanks very much.

**Comment by ggarra13** [comment]
I'm not too familiar with PDPlayer, as I've used it only in v1.5 or so when it did not support Open Color IO.  If you were using a similar version to that, it may not be possible to make it match as PDplayer used custom LUTs.  If you are using a more recent version of PD player which supports OCIO, it should be possible to make it match if you use the same config file and settings.

mrv2 uses OpenColorIO 2 (www.opencolorio.org).  I'm not a color expert so you may want to check the OCIO forums or mailing lists for most information and to corroborate the brief explanations below.

The process of configuring mrv2 for OpenEXRs is to first select a config file which you do by going to Window->Preferences->OCIO (by default mrv2 uses the cg-config file which is fine for most artists use and assumes an ACES workflow).  Then you select your display and view setiings for your monitor in the topbar pulldown of the main GUI  (the default of sRGB Display/ACES 1.0 - SDR Video should be fine for most uses too).

The final element you need to select is the input color space for your EXRs from the center pulldown of the top bar.  This is the main thing to concentrate on.  Depending on where and how you created your EXRs this may vary.  Most 3D applications now support OCIO so you should set it to match what your 3D application used to create your images.  If you are working in an ACES workflow, this will likely be ACEScg.  If you were not using ACES, you might try to see if you can make it match by using scene_linear from the Input Color Space->ACES->scene_linear submenut.

If your EXR images came from a camera, it may be needed to know if they were using ACES and then you should set it to what they used it to shoot to.  If the shoot it without taking ACES into account, the plates might be in the raw format of the camera, in which case you should set the input color space to one that closely matches the camera (one of the log one LUTs for example which are not in the cg-config but in the studio OCIO config file which you may need to download from the opencolorio.org web site).

Finally, if your EXRs are textures from a paint application, you may again need to find out if they were created in some OCIO color space, or in sRGB.  If sRGB you should use Utility->Texture->sRGB or similar.

Again, for more information I recommend you check with the OpenColorIO folks which aer truly color experts.

**Reply by KenTanker0us** [reply]
Gonzalo, thanks very much for your generosity, and your helpful and timely response. I will do my best.

**Reply by BigRoy** [reply]
@KenTanker0us were you able to get the colors the way you wanted them? If not, could you share previews of the differences (e.g. screenshots) between PDplayer and mrv2 and maybe some of the preferences/settings from PDplayer related to the color management so we could maybe try and further help in debugging the differences you're looking for.

**Reply by KenTanker0us** [reply]
I'm sorry I can't help with any useful information. I was directed to try cineSync Play and it plays exr sequences for me with the very same color I had with PDP; so I didn't really look into it further. I thank you for mrbv2 and I hope you get the info you require.
